apitools/gen/gen_client_lib.py

```diff
@@ -164,6 +170,73 @@ def WriteInit(self, out):
         printer()
         printer('__path__ = pkgutil.extend_path(__path__, __name__)')
 
+    def WriteIntermediateInit(self, out):
+        """Write a simple __init__.py for an intermediate directory."""
+        printer = self._GetPrinter(out)
+        printer('#!/usr/bin/env python')
+        printer('"""Shared __init__.py for apitools."""')
+        printer()
+        printer('from pkgutil import extend_path')
+        printer('__path__ = extend_path(__path__, __name__)')
```

Comment:
yep! shamelessly stolen from [the docs](https://docs.python.org/2/library/pkgutil.html#pkgutil.extend_path). :grin: